lib/server/context.js

```diff
@@ -99,7 +77,10 @@ Context.prototype.processPublication = function(publishHandler, publishContext,
     Meteor.setTimeout(function() {
       if (!future.isResolved()) {
         //publish handler failed to send ready signal in time
-        console.warn('Publish handler for', publishContext._subscription, 'sent no ready signal');
+        //maybe your non-universal publish handler is not calling this.ready()?
+        //or maybe it is returning null to signal empty publish?
+        //it should still call this.ready() or return an empty array []
```

Comment:
Feel free to modify it in any way you prefer. I like to keep logs short, but it is your package. :-)